yarn.lock

```diff
@@ -142,11 +142,23 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
+"@babel/code-frame@^7.14.5":
```

Comment:
Ideally this should deduplicated, but calling `yarn-deduplicate` regenerates the whole `yarn.lock`.

Reply:
`yarn-deduplicate` now has a `--scopes` arg, so hopefully `yarn-deduplicate --scopes @babel` would fix this (I haven't tried it yet).

Reply:
@ecraig12345 noice 🤗 Applied in cabae17